KGP.TicketApp.Backend/Helpers/JwtTokenHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Azure.Core;
+using KGP.TicketApp.Backend.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace KGP.TicketApp.Backend.Helpers
+{
+    public static class JwtTokenHelper
+    {
+        public static string CreateToken(string email, string id, ApplicationOptions settings)
+        {
+            var tokenDescriptor = new SecurityTokenDescriptor
+            {
+                Subject = new ClaimsIdentity(new[]
+                {
+                    new Claim("Id", Guid.NewGuid().ToString()),
+                    new Claim(JwtRegisteredClaimNames.Sub, email),
+                    new Claim(JwtRegisteredClaimNames.Email, email),
+                    new Claim(JwtRegisteredClaimNames.Jti,
+                    Guid.NewGuid().ToString())
+                }),
+                Expires = DateTime.UtcNow.AddMinutes(30),
+                Issuer = settings.JwtIssuer,
+                Audience = id,
+                SigningCredentials = new SigningCredentials(new SymmetricSecurityKey(Encoding.ASCII.GetBytes(settings.JwtKey)), SecurityAlgorithms.HmacSha512Signature)
+            };
+
+            var tokenHandler = new JwtSecurityTokenHandler();
+            var token = tokenHandler.CreateToken(tokenDescriptor);
+            var stringToken = tokenHandler.WriteToken(token);
+
+            return stringToken;
+        }
+        public static bool IsIdValid(string JwtToken, string id)
```

Comment:
tego gdziś używamy? i czemu id to string?

Reply:
Bo skonwertowałem wcześniej, muszę albo tutaj albo w wywołaniu funkcji bo zhaszowane id też jest stringiem

Reply:
oki, to zróbmy nazwę hashedId.